src/js/jsx/sections/libraries/LibraryList.jsx

```diff
@@ -153,12 +153,12 @@ define(function (require, exports, module) {
 
                 options = options.concat([
                     {
-                        title: strings.LIBRARIES.RENAME_LIBRARY.replace("%s", selectedLibrary.name),
```

Comment:
I love the new title, but we have the localized titles that still have the string tags. Maybe keep these replacement code until we confirm the translations have updated? @iwehrman 

https://github.com/adobe-photoshop/spaces-design/blob/master/src/nls/fr/strings.json#L472